client/src/EditorAssist.ts

```diff
@@ -9,6 +8,8 @@ export function unload() {
   subscriptions = [];
 }
 
+export const regExpQualifiedCoqIdent = /((\p{L}|_\u00A0)(\p{L}|[0-9_\u00A0'])*)(\.((\p{L}|_\u00A0)(\p{L}|[0-9_\u00A0'])*))*/u;
```

Comment:
@fakusb For some reason, Cmd-A on `_at_empSP_fwd.` gives `at_empSP_fwd not a defined object.` — ditto for `_at_empSP'_fwd` and `at_empSP'_fwd not a defined object.`

Reply:
Yes, that was an error, the () around `_\u00A0` was missing
fixed in c4878eabd806af64bb1933b1b723232f35670c40 , included in #175

Reply:
ah, I need a newer version of your package, or @maximedenes should just make you maintainer and we could crowdsource the review